src/components/common/ServicePart.tsx

```diff
@@ -7,7 +7,7 @@ interface ServiceSectionProps {
   highlights: string[];
 }
 
-const ServiceSection: React.FC<ServiceSectionProps> = ({
+const ServicePart: React.FC<ServiceSectionProps> = ({
```

Comment:
please rename the interface as well
you forgot to update ServiceSectionProps

Reply:
I guess you can name this file ServiceHighlights
Tbh servicepart name is a bit confusing